src/UI/class-settings-page.php

```diff
@@ -751,27 +762,43 @@ public function print_select_tag( array $args ): void {
 	}
 
 	/**
-	 * Prints out the radio buttons.
+	 * Prints the radio buttons.
 	 *
 	 * @param array $args The arguments for the radio buttons.
 	 */
-	public function print_binary_radio_tag( array $args ): void {
-		$options = $this->parsely->get_options();
-		$name    = $args['option_key'];
-		$value   = $options[ $name ];
-		$id      = esc_attr( $name );
-		$name    = Parsely::OPTIONS_KEY . "[$id]";
+	public function print_radio_tags( array $args ): void {
+		$name          = $args['option_key'];
+		$id            = esc_attr( $name );
+		$input_name    = Parsely::OPTIONS_KEY . "[$id]";
+		$options       = $this->parsely->get_options();
+		$selected      = $options[ $name ];
+		$radio_options = $args['radio_options'] ?? array(
+			'true'  => __( 'Yes', 'wp-parsely' ),
+			'false' => __( 'No', 'wp-parsely' ),
```

Comment:
These are present for `AMP` setting field, will updated it after getting feedback from @danielabloch